README.md

```diff
@@ -48,19 +100,10 @@ One of the ways to patch scikit-learn is by modifying the code. First, you impor
     with config_context(target_offload="gpu:0"):
         clustering = DBSCAN(eps=3, min_samples=2).fit(X)
     ```
+:eyes: Check out available [notebooks](https://github.com/intel/scikit-learn-intelex/tree/master/examples/notebooks) for more examples.
 
-👀 Read about [other ways to patch scikit-learn](https://intel.github.io/scikit-learn-intelex/index.html#usage) and [other methods for offloading to GPU devices](https://intel.github.io/scikit-learn-intelex/latest/oneapi-gpu.html).
-Check out available [notebooks](https://github.com/intel/scikit-learn-intelex/tree/master/examples/notebooks) for more examples.
-
-This software acceleration is achieved through the use of vector instructions, IA hardware-specific memory optimizations, threading, and optimizations for all upcoming Intel platforms at launch time.
-
-## Supported Algorithms
 
-❗ The patching only affects [selected algorithms and their parameters](https://intel.github.io/scikit-learn-intelex/latest/algorithms.html).
-
-You may still use algorithms and parameters not supported by Intel(R) Extension for Scikit-learn in your code. You will not get an error if you do this. When you use algorithms or parameters not supported by the extension, the package fallbacks into original stock version of scikit-learn.
-
-## 🚀 Acceleration
+## Acceleration
 
 ![](https://raw.githubusercontent.com/intel/scikit-learn-intelex/master/doc/sources/_static/scikit-learn-acceleration-2021.2.3.PNG)
```

Comment:
let's update this with SPR data that @ahuber21 done some time ago.

Reply:
where can i find it?

Reply:
Check with @ahuber21  please

Reply:
@napetrov thank you for pinging me. Will follow up offline. There are official charts for SPR data which we can add.

Reply:
@napetrov @ahuber21 i have updated the image, it may not be directly visible in the PR since it is a direct link, but it should be fine after it is merged